### PR TITLE
Fix Numba versions

### PR DESCRIPTION
**Pull request by awolant**
Signed-off-by: Albert Wolant <awolant@nvidia.com>

## Category:

Bug fix

## Description:

New realese of Numba is not compatible with how we test DALI in CI. It strictly requires CUDA 10.2 and our CI does not guarantee it.
Next release of Numba will drop Python 3.7 support, so I pinned Numba version for CUDA 11 as well for now.

This is experimental feature, so I decided to not include any info about this in the docs for now - we might want to pin another version in the future and now we do it due to the tests in CI only, so I think it is ok to not committing to any Number version for now.

### Tests:

- [x] Existing tests apply
- [ ] New tests added
  - [ ] Python tests
  - [ ] GTests
  - [ ] Benchmark
  - [ ] Other
- [ ] N/A

### Documentation
- [x] Existing documentation applies
- [ ] Documentation updated
  - [ ] Docstring
  - [ ] Doxygen
  - [ ] RST
  - [ ] Jupyter
  - [ ] Other
- [ ] N/A

#### Requirements
- [ ] Implements new requirements
- [ ] Affects existing requirements
- [ ] N/A

**REQ IDs**: N/A

**JIRA TASK**: N/A

### CODE REVIEW

**Review by klecki — APPROVED** [review]

File: qa/setup_packages.py [hunk 1]
```diff
@@ -453,7 +453,14 @@ def get_pyvers_name(self, url, cuda_version):
                         { "100" : [
                             PckgVer("2.2.0", dependencies=["protobuf<4"])],
                           "110" : [
-                            PckgVer("2.2.0", dependencies=["protobuf<4"])]})
+                            PckgVer("2.2.0", dependencies=["protobuf<4"])]}),
+                CudaPackage("numba",
+                        { "100" : [
+                            PckgVer("0.55.2", python_min_ver="3.7"),
+                            PckgVer("0.53.1")],
```
Comment (JanuszL) [inline qa/setup_packages.py:460]:
```suggestion
                            PckgVer("0.53.1",  python_max_ver="3.6")],
```

File: qa/setup_packages.py [hunk 2]
```diff
@@ -453,7 +453,14 @@ def get_pyvers_name(self, url, cuda_version):
...
+                          "110" : [
+                            PckgVer("0.56.0", python_min_ver="3.7"),
+                            PckgVer("0.53.1")]})
```
Comment (JanuszL) [inline qa/setup_packages.py:463]:
```suggestion
                            PckgVer("0.53.1",  python_max_ver="3.6")]})
```

File: qa/setup_packages.py (hunk 1 above)
Comment (JanuszL) [reply qa/setup_packages.py:460]: Otherwise we will test  0.55.2 and 0.53.1 for pythons 3.7 and above.

Comment (awolant) [reply qa/setup_packages.py]: Done. I had that and removed it before :)

File: qa/setup_packages.py (hunk 2 above)
Comment (awolant) [reply qa/setup_packages.py]: Done

**Review by JanuszL — APPROVED** [review]

**Pull request merged**